2021q4/office.adoc

```diff
@@ -0,0 +1,26 @@
+=== FreeBSD Office team 2021Q4 status report
+
+Links: +
+link:https://wiki.freebsd.org/Office[The FreeBSD Office project] URL: link:https://wiki.freebsd.org/Office[https://wiki.freebsd.org/Office]
+
+Contact: FreeBSD Office team ML <office@FreeBSD.org> +
+Contact: Dima Panov <fluffy@FreeBSD.org> +
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org>
+
+The FreeBSD Office team works on a number of office-related software suites and tools such as OpenOffice and LibreOffice.
+
+Work during this quarter was focused on providing the latest stable release of LibreOffice suite and companion apps to all FreeBSD users.
+
+Latest and quarterly ports branches got a new branch (7.2) of the LibreOffice suite and updated to 7.2.4 release while new preleases
+such as 7.2.5.RC2 and 7.3.0.RC1 are cooking in WIP stage area.
+
+Meanwhile, our link:https://github.org/freebsd/freebsd-ports-libreoffice[WIP repository] got back a working CI instance again, thanks to Li-Wen Hsu.
+
+Also we still working on link:https://github.com/fluffykhv/freebsd-ports-boost[Boost WIP repositiry] to bring latest Boost library to the ports.
```

Comment:
s/we/we are/

s/Boost WIP/the Boost WIP/

s/repositiry/repository/ (typo)

s/latest/the latest/